Sources/SwiftDocC/Infrastructure/ConvertActionConverter.swift

```diff
@@ -0,0 +1,203 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+package enum ConvertActionConverter {
```

Comment:
> Why is this distinct from `ConvertAction`? 

Mainly because this code uses some types and API that are internal to `SwiftDocC`.  `ConvertAction` is defined in a different module and can't access that internal API. We could make the necessary API `package` level, but I thought that it would be a smaller change to keep that code in `SwiftDocC` and let `ConvertAction` call it.

> Also, why is this an enum? I don't see any enum cases defined below anywhere. This seems like a Swift idiom I'm not familiar with.

This pattern is sometimes used as a namespace in Swift. The reason for using an enum for this is that enum without any cases is "uninhabited" and can't be initialized which means that callers can only ever access its static API which limits the possibilities for state and mutation. 

In this case the enum only has a single static function so the "namespace" doesn't provide much benefit compared to just a global function and since it's only `package` access we can change it freely in the future without needing to make deprecations.